src/deterministic.js

```diff
@@ -9,9 +9,18 @@ Math.random = () => {
   MAGIC = Math.pow(MAGIC + 1.8912, 3) % 1;
   return MAGIC;
 };
+
 var TIME = 10000;
-Date.now = () => TIME++;
-if (typeof performance == 'object') performance.now = Date.now;
+function deterministicNow() {
+  return TIME++;
+}
+
+Date.now = deterministicNow;
+
+// Setting performance.now to deterministicNow doesn't work so we instead
+// use a helper function in parseTools (getPerformanceNow()) to call it
+// directly.
```

Comment:
Maybe link to this PR as a source for the issue?

Reply:
I'm not sure thats really necessary.. isn't that what `git blame` is for?